### ISSUE TITLE
Suggestion: add a switcher to show/hide _id,_key_,_rev

### ISSUE THREAD

**Issue by AlexLuya** [arangodb/arangodb#12531]
![Workspace 1_987](https://user-images.githubusercontent.com/303353/91244779-cf1f0180-e77f-11ea-8da6-8a3585a1bdf1.png)

Most of time,user won't care about these these item,better to add a switcher to hide/show them

**Comment by dothebart** [comment]
you're right, at least either `_id` or `_key` could be hidden, `_rev` is probably uninteresting for most.

**Comment by lostpebble** [comment]
Yea, I'm actually realising how much of an annoyance this is now as well. We already have the `_key` column, we already know what collection we're in (no need for `_id`) and `_rev` is an internal value which doesn't tell us much in the dashboard.

All those three should be removed from the "preview" of the content. Even when you click through to a document, these values don't show inside the actual data structure- only at the top (which is good).

Another request I would have is the ability to make the font size smaller for the preview as well- this will allow us to see more of the content, depending on how small we're willing to go. Having the ability to quickly see which piece of content you're looking for is a great dev experience.

**Comment by 153957** [comment]
> you're right, at least either `_id` or `_key` could be hidden, `_rev` is probably uninteresting for most.

Both `_id` and `_key` can be hidden. The `_key` is shown in a separate column, and the collection name (additional part of the `_id`) is shown at the top of the page.

**Comment by 153957** [comment]
We are using encoded uuid4 strings as unique object ids, which means that both the `_id` and `_key` are very long and only by zooming out can we sometimes see one of the field values. Please fix this.

**Comment by 153957** [comment]
I would actually like to request an option to hide all 'internal' attributes, similar to the `removeInteral` option for the [`ATTRIBUTES`](https://www.arangodb.com/docs/devel/aql/functions-document.html#attributes) function,
because we also add some custom internal attributes which are uninteresting to show in that overview.

**Comment by 153957** [comment]
Should this be changed here? https://github.com/arangodb/arangodb/blob/devel/js/apps/system/_admin/aardvark/APP/frontend/js/collections/arangoDocuments.js#L220
